pulpcore/cli/rpm/content.py

```diff
@@ -108,10 +111,18 @@ def content() -> None:
     exclude_field_option,
     pulp_option("--repository-version"),
     pulp_option("--arch", allowed_with_contexts=(PulpRpmPackageContext,)),
+    pulp_option(
+        "--arch-contains", "arch__contains", allowed_with_contexts=(PulpRpmPackageContext,),
+        needs_plugins=[PluginRequirement("rpm", specifier=">=3.20.0")],
```

Comment:
At this point i would say the guard in pulp-glue should be sufficient, no?

Also think about adding a `feature` parameter.

Reply:
I personally quite like them to be in two places, saying that however, I am not that familiar with the Pulp code base, so I'd rely on you to let me know the best practice here.

Yeah, I was thinking about the feature parameter, but deleted it from the pulp-glue preprocess function since I prefer the any function for the conditional check instead of a full loop. If we are going to add feature values, can we add them to the individual call in options as highlighted?

Reply:
We could call it something like "substring filters".

Reply:
updated as suggested, please check. thanks.